FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Minimal_Windows_Simulator/main.c

```diff
@@ -185,14 +186,15 @@ int main( void )
         }
         #endif /* ( ipconfigUSE_DHCP != 0 ) */
 
-        memcpy( ipLOCAL_MAC_ADDRESS, ucMACAddress, sizeof( ucMACAddress ) );
-
-        FreeRTOS_IPInit_Multi();
+        /*memcpy( ipLOCAL_MAC_ADDRESS, ucMACAddress, sizeof( ucMACAddress ) ); */
```

Comment:
Can we just remove this line instead of comment out?

Reply:
updated.